### PR TITLE
Add Friendica console part 2: Script removal

### PR DESCRIPTION
**Pull request by MrPetovan**
Part of #4629

To be reviewed after #4632

### CODE REVIEW

**Comment by tobiasd** [comment]
Are those scrips still callable without the console? Also github merged all the PRs it seems, making it hard to look into the code changes.

File: CHANGELOG [hunk 1]
```diff
@@ -485,7 +485,7 @@ Version 3.3.3 (2015-02-24)
 	Share-it button support (annando)
 	More reliable reshare from Diaspora (annando)
 	Load more images via proxy (annando)
-	util/typo.php uses "php -l" insead of "eval()" to validate code (fabrixxm)
```
Comment (tobiasd) [inline CHANGELOG]: please revert this change! Historc documents shall not be altered in retrospect, but only with a new entry labeled witht eh new version/date tag.

File: index.php [hunk 2]
```diff
@@ -51,9 +51,13 @@
 require_once "include/dba.php";
 
 if (!$install) {
-	dba::connect($db_host, $db_user, $db_pass, $db_data, $install);
+	$result = dba::connect($db_host, $db_user, $db_pass, $db_data);
```
Comment (tobiasd) [inline index.php]: This is just another merge conflict your are summoning? Or is there a reason to drop the `install` parameter?

Comment (annando) [reply index.php]: I stumbled upon it as well, but it is okay. The "connect" now always returns "false" on an error. The later code then have to handle it by throwing this "system unavailable" message.

I hadn't checked by now if the return value is checked now on every place where there is a "connect". This - of course - had to be done as well.

**Comment by MrPetovan** [comment]
> Are those scrips still callable without the console?

No, but each subconsole can be given its own launcher and work independently of the main console.

> Also github merged all the PRs it seems, making it hard to look into the code changes.

I'll break it down in two parts, the code add, and the code removal, it is different from the other one.

File: index.php (hunk 2 above)
Comment (MrPetovan) [reply index.php]: In this PR?

Comment (annando) [reply index.php]: You removed this "unavailable"-message in this PR. So it should be added to all places where the "connect" is called (except the installer of course).

Comment (MrPetovan) [reply index.php]: Fair enough.

**Comment by tobiasd** [comment]
> No, but each subconsole can be given its own launcher and work
> independently of the main console.

Why not stick with KISS principle here and make it more complicated?

I know that Friendica itself does not really follow KISS, but at least we can keep our helper script follow the principle.

**Comment by MrPetovan** [comment]
To be clear, there is absolutely no good reason to create separate launchers for the individual console commands. But it still is possible, the code is modular enough to allow it.

I had four main reasons for creating this console:
- Have fun
- Have a fancy console like popular frameworks
- Remove PHP scripts from the `util` folder
- Normalize the PHP scripts to get an unified feeling using them.

I don’t care much about KISS as is because it isn’t the governing principle in Friendica like you accurately remarked, and because it is highly subjective, there aren’t any standard for KISS, which means my idea of KISS wouldn’t be the same as yours, which introduces communication issues, ultimately defeated the general purpose of KISS.

You don’t like the changes I introduced, it’s fine, but mentioning the loose KISS principle isn’t actionable for me, because I could feel my changes already follow my idea of KISS.

**Comment by tobiasd** [comment]
Ok, so we are having different ideas of KISS. That is fine. Have fun with your console.
